src/common/translation/i18n/fi.json

```diff
@@ -330,6 +330,20 @@
     "searchCategories": "Selaa kategorioita",
     "searchEvents": "Etsi tekemistä"
   },
+  "meta":{
+    "events": {
+      "description": "Tapahtumat.hel.fin tapahtumahaku. Konsertit, keikat, teatteriesitykset, näyttelyt, kulttuuri, tekeminen ja muut tapahtumat Helsingissä."
+    },
+    "courses": {
+      "description": "Tapahtumat.hel.fin harrastushaku. Lasten ja nuorten harrastukset, kurssit, kerhot ja leirit Helsingissä."
+    },
+    "collections": {
+      "description": "Tutustu suosituksiimme tämän hetken ajankohtaisimmista ja kiinnostavimmista tapahtumista."
+    },
+    "default": {
```

Comment:
Could find these `default` texts from the jira-ticket but perhaps these texts were determined somewhere else?

Reply:
in translations only 3 landing pages for /courses /events and /collections. but we have /home as well, so i kept the same translations as "default" for that.